platform-hub-api/spec/controllers/kubernetes/clusters_controller_spec.rb

```diff
@@ -218,6 +224,8 @@
           expect(updated.aws_account_id).to eq put_data[:aws_account_id].to_i
           expect(updated.s3_region).to eq put_data[:s3_region]
           expect(updated.s3_bucket_name).to eq @cluster.s3_bucket_name
+          expect(updated.api_url).to eq put_data[:api_url]
+          expect(updated.ca_cert_encoded).to eq put_data[:ca_cert_encoded]
```

Comment:
`ca_cert_encoded` is unset in the `put_data` above, is this valid?

Reply:
Oh nice catch – this line should be:

```
expect(updated.ca_cert_encoded).to eq @cluster.ca_cert_encoded
```

(note that the PUT operation is additive, in that it won't unset any fields that are not explicitly provided)